kube/rbac.go

```diff
@@ -52,15 +52,16 @@ func NewRBACAccount(name string, account model.AuthAccount, settings ExportSetti
 	}
 
 	binding := newTypeMeta("rbac.authorization.k8s.io/v1", "ClusterRoleBinding", blockPSP)
-	binding.Add("metadata", helm.NewMapping("name", fmt.Sprintf("%s-binding-psp", name)))
+	binding.Add("metadata", helm.NewMapping("name",
+		authCRBindingName(name, settings.CreateHelmChart)))
```

Comment:
We generally try to pass just `settings` into the lower level functions, and let them check for `settings.CreateHelmChart` themselves.

Reply:
Addressed in commit 8f8210a40edcde2491a1d2e44f13d68b1fefaa27